BuildLoader.py

```diff
@@ -1076,6 +1077,9 @@ def post_build(self):
 		# create bootloader reserved binary of 4K size
 		gen_file_with_size (os.path.join(self._fv_dir, 'SBLRSVD.bin'), 0x1000)
 
+		# ccreate variable region for Epayload
+		if self._board.EPVARIABLE_SIZE > 0:
```

Comment:
Maybe we could also check if UEFI payload is built-in later. ideally if UEFI payload is not built-in, we should not build a UEFI variable region.